### PR TITLE
Implement new forcing in marrmot class

### LINKED ISSUE #92: Lisflood model class still uses old `forcing_data`
This breaks the package as soon as you do `from ewatercycle.models import something`.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
closes #92  #91

### CODE REVIEW

File: examples/MarrmotM01.ipynb [hunk 1]
```diff
@@ -34,7 +34,7 @@
   {
```
Comment (sverhoeven) [inline examples/MarrmotM01.ipynb:34]: <p>Typo in heading 3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: ewatercycle/models/marrmot.py [hunk 2]
```diff
@@ -261,11 +254,8 @@ def parameters(self) -> Iterable[Tuple[str, Any]]:
             ('maximum_soil_moisture_storage', self._parameters[0]),
             ('initial_soil_moisture_storage', self.store_ini[0]),
             ('solver', self.solver),
-            ('start time', self.forcing_start_time.isoformat()),
-            ('end time', self.forcing_end_time.isoformat()),
+            ('start time', self.forcing_start_time.strftime("%Y-%m-%dT%H:%M:%SZ")),
```
Comment (sverhoeven) [inline ewatercycle/models/marrmot.py:257]: Looks a bit weird to have a format string, however to get UTC timezone this is needed. Also `self.forcing_*_time` gets sets using get_time which forces UTC so OK to hard-code timezone here.

File: tests/models/test_marrmotm01.py [hunk 3]
```diff
@@ -25,8 +26,19 @@ def forcing_file(self, sample_marrmot_forcing_file):
...
-    def model(self, forcing_file: Path, mocked_config):
-        m = MarrmotM01(version="2020.11", forcing=forcing_file)
+    def generate_forcing(self, forcing_file):
+        forcing = load_foreign('marrmot',
+                                directory=str(Path(forcing_file).parent),
+                                start_time='1989-01-01T00:00:00Z',
+                                end_time='1992-12-31T00:00:00Z',
+                                forcing_info={
+                                    'forcing_file': str(Path(forcing_file).name)
+                                })
```
Comment (sverhoeven) [inline tests/models/test_marrmotm01.py]: Indent is one space to many
```suggestion
                               directory=str(Path(forcing_file).parent),
                               start_time='1989-01-01T00:00:00Z',
                               end_time='1992-12-31T00:00:00Z',
                               forcing_info={
                                   'forcing_file': str(Path(forcing_file).name)
                               })
```

**Review by sverhoeven — APPROVED** [review]
After commenting out Lisflood in models/__init__py the tests ran OK.

Notebook also ran OK.

Some small typos, style points, but overall nice work.

**Comment by SarahAlidoost** [comment]
> After commenting out Lisflood in models/__init__py the tests ran OK.
> 
> Notebook also ran OK.
> 
> Some small typos, style points, but overall nice work.

Many thanks for reviewing, I addressed your comments.  Shall I merge this PR?

**Pull request merged**